x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -17,6 +17,7 @@ import {
   EndpointStatus,
 } from './types';
 import { factory as policyFactory } from './models/policy_config';
+import { parentEntityId } from './models/event';
```

Comment:
These changes are in this PR: #70129

they will be removed once the other PR gets merged.